libraries/provision/install_sync_gateway.py

```diff
@@ -61,6 +61,9 @@ def sync_gateway_base_url_and_package(self, sg_ce=False, sg_platform="centos", s
             if sg_ce:
                 sg_type = "community"
 
+            if (sg_platform == "windows" or sa_platform == "windows") and (sg_installer_type != "msi" or sa_installer_type != "msi"):
+                platform_extension["windows"] = "exe"
```

Comment:
Is this because versions of 1.1.0 and 1.1.1 does not have msi support, you added platform extension as exe for sg_installer_type is msi?

Reply:
Version 1.x is not even supported for provisioning